config/openvpn-client-export/openvpn-client-export.inc

```diff
@@ -44,20 +44,17 @@ $current_openvpn_version_rev = "03";
 
 function openvpn_client_export_install() {
 	global $current_openvpn_version;
+	conf_mount_rw();
+	$tarpath = "/usr/local/pkg/openvpn-client-export-{$current_openvpn_version}.tgz";
+	$phpfile = "vpn_openvpn_export.php";
```

Comment:
Err... that's actually a mistake. My bad - github newbie here. I think this is because my version of the file is based on what's currently found on a live pfsense box, and it looks as if the git master branch has a newer version. I'll update my branch with an amendment, which (I believe) should update this pull request.